src/finch/io.py

```diff
@@ -0,0 +1,11 @@
+from .julia import jl
+from .tensor import Tensor
+
+
+def read(filename: str) -> Tensor:
+    julia_obj = jl.fread(filename)
+    return Tensor(julia_obj)
+
+
+def write(filename: str, tns: Tensor) -> None:
+    jl.fwrite(filename, tns._obj)
```

Comment:
Same here, `pathlib.Path` support would be nice.

Reply:
Done!